### DISCUSSION TITLE
Unable to add Commit message as I want

### DISCUSSION THREAD

**Discussion by Suvadip-sana** [recodehive/Stackoverflow-Analysis#370] [General]
![Screenshot 2024-10-02 190242](https://github.com/user-attachments/assets/34ca3665-c5ef-48fc-ad07-26f1ce36f1c5)

I recent commit in open source with that commit message (highlighted in red marker). But I want that type of commit message (highlighted in blue marker) where it has a issue number along with Github user ID who raised this.

**How a add this type of commit message when committing in opensource project?**

Please someone help me sir @sanjay-kv !

**Comment by sanjay-kv** [comment]
thats not possible. below is standard structure github follows. 

if you need to add issue number that you need to spesficy while you commit. commit message are there to show the version history with comment. the other one you talking about is merge PR both are different
https://www.youtube.com/watch?v=QlTrHXmr-JI&list=PLrLTYhoDFx-kiuFiGQqVpYYZ56pIhUW63&index=9

watch this video

![image](https://github.com/user-attachments/assets/e2e57573-90a9-4c6f-b60c-d8e6438d36c6)
